tests/python/relay/aot/test_crt_aot.py

```diff
@@ -43,7 +43,8 @@
 from aot_test_utils import *
 
 
-def test_conv_with_params():
+@pytest.mark.parametrize("target_options", ["", "--typed-operators=0"])
```

Comment:
I am worried about test explosion. Once you rebase and incorporate @manupa-arm changes (i.e., memory workspace feature) each test would run now 4 times [with/without workspace] x [with/without typed-operators]. Maybe you could select some tests where enabling this feature? Or maybe enable it by default, and add few tests without the feature?

Reply:
I'm going to leave the smaller tests with all permutations as they're fast. The worrisome one is Mobilenet so I'll default that to having `typed-operators=0` and the memory tests?